www/docs/en/dev/config_ref/index.md

```diff
@@ -470,6 +470,23 @@ Examples:
 <hook type="after_plugin_install" src="scripts/afterPluginInstall.js" />
 ```
 
+## resource-file
+
+This tag installs resource files into your platform, and is similar to the same tag in plugin.xml. This tag is currently only supported on `cordova-ios@4.4.0` or greater and `cordova-android@6.2.1` or greater.
+
+Attributes(type) <br/> <span class="sub-header">Only for platform:</span> | Description
+---------------- | ------------
+src(string) <br/> ==iOS== ==Android==| *Required* <br/> Location of the file relative to `plugin.xml`. If the src file can't be found, the CLI stops and reverses the installation, issues a notification about the problem, and exits with a non-zero code.
```

Comment:
The description consider the context to be of a plugin, which is fortunately not the case anymore.